src/logic/player.h

```diff
@@ -725,7 +725,9 @@ class Player {
 	uint8_t initialization_index_;
 	bool see_all_{false};
 	bool random_tribe_{false};
-	StartingPositionState starting_position_state_{StartingPositionState::kFinal};
+	bool is_picking_custom_starting_position_{false};
+	// Only for the local player
+	bool is_setting_custom_starting_position_{false};
```

Comment:
These names are too similar with only 4 out of 36 characters differing, so they can easily be confused with each other. Perhaps rename the second one `local_player_is_selecting_custom_starting_position_` or such.